zcash_client_sqlite/src/wallet/db.rs

```diff
@@ -1084,6 +1088,12 @@ WITH unioned AS (
     -- join on the accounts table to obtain account UUIDs
     LEFT JOIN accounts from_account ON from_account.id = sent_notes.from_account_id
     LEFT JOIN accounts to_account ON to_account.id = ro.account_id
+    -- join to transparent_received_output_spends to gather data about transaction inputs
+    LEFT OUTER JOIN transparent_received_output_spends ros
+       ON ros.transaction_id = t.id_tx
```

Comment:
Bug: `t` is undefined. Presumably it is meant to be:
```suggestion
       ON ros.transaction_id = transactions.id_tx
```